app-gateway.tf

```diff
@@ -1,6 +1,6 @@
 data "azurerm_key_vault" "infra_vault" {
-  name = "infra-vault-${var.subscription}"
-  resource_group_name = "${var.env == "prod" ? "core-infra-prod" : "cnp-core-infra"}"
+  name = "${var.env == "prod" ? "infra-cert-prod" : "infra-vault-${var.subscription}"}"
```

Comment:
I am sure these changes are not part of this PR ? could you please rebase ?

Reply:
hmm.. let me actually rebase it. i did the merge locally, not sure why this tf file got screwed. [never merge - just rebase :D]